.github/BOTMETA.yml

```diff
@@ -1031,7 +1031,7 @@ files:
     support: network
     maintainers: $team_networking
   $plugins/cliconf/exos.py:
```

Comment:
Do you mind adding these lines at 1150.
 ```
 $plugins/httpapi/exos.py:
    maintainers: $team_extreme
```

Reply:
Done! Should have thought of that one earlier